src/video.rs

```diff
@@ -483,15 +482,15 @@ fn compare_videos_inner<D: Decoder + 'static, E: Decoder + 'static>(
     };
 
     let mut results = BTreeMap::new();
-    let mut avg = 0f64;
+    let mut rolling_mean = 0f64;
     for score in result_rx {
         if verbose {
             println!("Frame {}: {:.8}", score.0, score.1);
         }
 
         results.insert(score.0, score.1);
-        avg = avg + (score.1 - avg) / (min(results.len(), 10) as f64);
-        progress.set_message(format!(", avg: {:.1$}", avg, 2));
+        rolling_mean = rolling_mean + (score.1 - rolling_mean) / (results.len() as f64);
+        progress.set_message(format!(", mean: {:.1$}", rolling_mean, 2));
```

Comment:
It's a bit confusing to use a parameter for the width here (i.e. `:.1$`) when we can just use a constant `:.2`:

```suggestion
        progress.set_message(format!(", mean: {rolling_mean:.2}"));
```

Reply:
I didn't even know it was possible this way, I should really read the rust docs on formatting strings...